openedx/features/qverse_features/registration/templates/registration/edx_ace/registrationnotification/email/body.html

```diff
@@ -0,0 +1,58 @@
+{% extends 'ace_common/edx_ace/common/base_body.html' %}
+
+{% load i18n %}
+{% load ace %}
+{% load static %}
+{% block content %}
+<table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
+    <tr>
+        <td>
+            <h1>
+                {% blocktrans %}
+                    Hi {{firstname}},
+                {% endblocktrans %}
+            </h1>
+
+            <p style="color: rgba(0,0,0,.75);">
+                {% blocktrans %}
+                    You have been registered successfully on {{ site_name }}.
+                {% endblocktrans %}
+                <br />
+            </p>
+
+            <p style="color: rgba(0,0,0,.75);">
+                {% blocktrans %}
+                    This is your login information:
+                {% endblocktrans %}
+            </p>
+
+            <p style="color: rgba(0,0,0,.75);">
+                <b>{% blocktrans %}Registration Number (username):{% endblocktrans %}</b>
+                {% blocktrans %}{{ regno }}{% endblocktrans %}
+                <br />
+                <b>{% blocktrans %}Email:{% endblocktrans %}</b>
+                {% blocktrans %}{{ email }}{% endblocktrans %}
+                <br />
+                <b>{% blocktrans %}Password:{% endblocktrans %}</b>
+                {% blocktrans %}{{ regno }}{% endblocktrans %}
```

Comment:
i though we are using some random characters as a password, having regno as a password is a security risk, can we change this as part of this PR?

Reply:
done in this PR.